lib/lte_link_control/lte_lc.c

```diff
@@ -706,6 +707,8 @@ static int connect_lte(bool blocking)
 	if (err || !blocking) {
 		goto exit;
 	}
+	/* We do not really need to restore original functional mode before setting it to normal */
+	original_func_mode_valid = true;
```

Comment:
Just a suggestion, if the name of the variable would be for example `func_mode_changed`, the code would be self-explanatory, I think.

Reply:
Ok, I'll change it and remove the comment.